src/routes/users.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const express = require('express');
+
+const {
+  init,
+  getAll,
+  getUserById,
+  createUser,
+  removeUser,
+  updateUser,
+} = require('../services/users');
+
+const router = express.Router();
+
+function createUsers(app) {
+  app.use('/users', router);
+  init();
+
+  router.get('/', (req, res) => {
+    const users = getAll();
+
+    res.send(users);
+    res.statusCode = 200;
```

Comment:
it will be great to transfer logic from router, in controllers, you could create function
```
function getAll(req, res) {
   const users = usersServise.getAll();

   res.statusCode = 200;
   res.send(users);
 }
```
and your route will look like this
`router.get('/', usersControllers.getAll);`

you also need to change other routes